### ISSUE TITLE
The hose image should be shown where the water drops originate

### ISSUE THREAD

**Issue by samreid** [phetsims/wave-interference#93]
The hose image should be offset vertically compared to the other sources.  Kind of waiting on a decision on https://github.com/phetsims/wave-interference/issues/15#issuecomment-410761082 since it would impact the positioning.

**Comment by arouinfar** [comment]
@samreid just as a heads up, the team decided to iterate on the water source with the graphic artist. It will have similar dimensions to the hose, but it won't be the hose itself.

**Comment by samreid** [comment]
I updated the location of the hose image to match the water drops in #96.  I'll create a new issue for the preceding comment.
